src/xai_grok/__init__.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+import requests
+
+from xai_grok.grok import Grok as GrokClient
+import xai_grok.providers as providers
+
+
+def Grok(api_key: str = ""):
+    user_api_key = api_key if api_key else os.environ["XAI_API_KEY"]
+    return GrokClient(user_api_key, requests, providers.UnauthorizedUserErrorProvider())
```

Comment:
The function `Grok` uses `os.environ["XAI_API_KEY"]` without handling the case where the environment variable is not set, which will raise a `KeyError`. Consider using `os.environ.get("XAI_API_KEY")` with a default value or handling the exception.